src/wallet/wallet2_api.h

```diff
@@ -406,8 +473,20 @@ struct Wallet
     virtual ConnectionStatus connected() const = 0;
     virtual void setTrustedDaemon(bool arg) = 0;
     virtual bool trustedDaemon() const = 0;
-    virtual uint64_t balance() const = 0;
-    virtual uint64_t unlockedBalance() const = 0;
```

Comment:
@kenshi84 would it somehow be possible to keep the old address(), balance(), unlockedBalance() functions and make them use default account/address indices? I'm aware of https://github.com/monero-project/monero-core/pull/908, but there are also other wallets using this API (monerujo for example). @m2049r @luigi1111 @moneromooo-monero

Reply:
Whoaa!!! Guys, please don't kill API methods "just like that". Deprecate them and give us at least one release (better 2) time to catch up. Otherwise it's impossible to keep uninterrupted service. Also, please document deprecations clearly & prominently in the release notes!

Thanks for the ping @Jaqueeee .

Reply:
@Jaqueeee 
Sorry for making abrupt changes to the wallet API, and thank you for quickly making a patch #2703 .